### PR TITLE
chore: 修改生息演算不同模式的介绍

### PR DESCRIPTION
**Pull request by ABA2396**
I think I can group a few of you together, so you can be \@mentioned as a group. XD

cc @momomochi987 @Constrat @HX3N @Manicsteiner @wallsman 

@XuQingTW 如果你也愿意的话可以先加组织

## Summary by CodeRabbit

- **新功能**
	- 更新了游戏中的本地化文本，明确了在不同存档情况下获取点数的条件，改善了用户对游戏机制的理解。

- **文档**
	- 对多种语言的本地化文件进行了修改，提升了说明的清晰度和结构性，帮助用户更好地掌握游戏功能。

### CODE REVIEW

**Comment by Constrat** [comment]
Ai or not I still can't understand anything xD

**Comment by Constrat** [comment]
Is txwy considered international server?

**Comment by AnnAngela** [comment]
@Constrat Yes, we are testing CodeRabbit AI for quickly summarizing PR.

**Comment by ABA2396** [comment]
> Is txwy considered international server?

It's actually readable in simplified Chinese. It just might have a more authentic translation

**Comment by Constrat** [comment]
No no i meant for the @ in the team name xD

**Comment by ABA2396** [comment]
> Yes, we are testing CodeRabbit AI for quickly summarizing PR.

说实话这个总结我也看不懂）

**Comment by ABA2396** [comment]
> No no i meant for the @ in the team name xD

just @MaaAssistantArknights/global-support-team

**Comment by Constrat** [comment]
Is it correct it being a sub team?

**Comment by ABA2396** [comment]
No problem. This way you don't have to set separate permissions.

**Comment by Constrat** [comment]
Interesting 🤔

**Comment by XuQingTW** [comment]
我是可以來但我不知道怎麼加（

**Comment by ABA2396** [comment]
> 我是可以來但我不知道怎麼加（

看看邮箱，应该已经发了

**Comment by AnnAngela** [comment]
> > Yes, we are testing CodeRabbit AI for quickly summarizing PR.
> 
> 说实话这个总结我也看不懂）

i18n的PR估计啥AI的总结都不太行

**Comment by Constrat** [comment]
There's a lot of random stuff in a i18n PR that usually doesn't make sense from a coding / logical point of view no?

**Comment by ABA2396** [comment]
I really don't know anything about Korean. Maybe we need more friends

或许已经是时候去b站招人了 😋

File: src/MaaWpfGui/Res/Localizations/en-us.xaml [hunk 1]
```diff
@@ -846,15 +846,15 @@ The video aspect ratio needs to be 16:9 without interference factors such as bla
...
-  3. Can only be started at the main nav page of reclamation algorithm. 
+  3. Can only be started at the main nav page of reclamation algorithm.
 
-To get points by manufacturing: 
-  1. it must be the first day after settlement, and there should be no enemy attacks entering the stationed area for the next three days. 
+With save file: farm points through crafting (advanced).
+  1. It must be the first day after settlement, and there should be no enemy attacks entering the stationed area for the next three days.
   2. It must start when entering the reclamation algorithm. (stationed location needs to be visible)
-  3. If the quantity that can be manufactured is exactly a multiple of 99, MAA can be confused (stuck). You can use up a little before saving. This might be repaired later. 
+  3. If the quantity that can be manufactured is exactly a multiple of 99, MAA can get confused (stuck). You can use up a little before saving. This might be repaired later.
```
Comment (Constrat) [inline src/MaaWpfGui/Res/Localizations/en-us.xaml:857]: @ABA2396 
1. By settlement we mean first day after "saving the file"?
2. Must start when entering RA? Stationed location visible? What's that?

**Comment by Constrat** [comment]
By the way, is it possible to get multi language support in coderabbit?

File: src/MaaWpfGui/Res/Localizations/en-us.xaml (hunk 1 above)
Comment (ABA2396) [reply src/MaaWpfGui/Res/Localizations/en-us.xaml:857]: 1. The first day after obtaining points (either completing a 3-day settlement or loading a saved file).
2. A place where you can see your base, which is within the map, different from the start screen without a saved file.

**Comment by Constrat** [comment]
Oh, so the team is made for translation purposes not for global feats?

**Comment by Constrat** [comment]
This guy yaps too much

**Comment by zkywalker230061** [comment]
Not sure if I can join the translation team... I can also help translate between CN andEN.

**Comment by ABA2396** [comment]
> Not sure if I can join the translation team... I can also help translate between CN andEN.

如果你想的话当然可以

**Comment by Constrat** [comment]
@zkywalker230061 The translations are available on webplate. If you want a quick GUI instewad of having to use VSCode to modify directly the xaml files

**Comment by ABA2396** [comment]
> @zkywalker230061 ~The translations are available on webplate. If you want a quick GUI instewad of having to use VSCode to modify directly the xaml files~ ~[weblate.maa-org.net/engage/maa](https://weblate.maa-org.net/engage/maa/)~ @ABA2396 I just noticed webplate is under maint? Was it _finally_ locked? XD

It will reports conflicts when main repository is a force push, so I say why don't do this on dev XD

**Comment by ABA2396** [comment]
> > @zkywalker230061 ~The translations are available on webplate. If you want a quick GUI instewad of having to use VSCode to modify directly the xaml files~ ~[weblate.maa-org.net/engage/maa](https://weblate.maa-org.net/engage/maa/)~ @ABA2396 I just noticed webplate is under maint? Was it _finally_ locked? XD
> 
> It will reports conflicts when main repository is a force push, so I say why don't do this on dev XD

Well, the server is indeed under maintenance.  XD

**Comment by Constrat** [comment]
But I noticed that force push on all my branches too, never understood where it actually came from. When I directly make changes to dev I always use VSCode GUI to sync, which does 
```
git pull --rebase origin dev
git push origin dev
```

**Comment by ABA2396** [comment]
> It will reports conflicts when main repository is a force push, so I say why don't do this on dev XD

image

**Comment by Constrat** [comment]
⭕ . ⭕

**Comment by ABA2396** [comment]
@Constrat It should be fine already. Try it?

**Comment by Constrat** [comment]
It's working now :)

**Comment by Constrat** [comment]
Something I'm still missing is that there are changes to the source file, webplate will still consider unfinished and I haven't found a way to tell him: we don't care it's correct skip it.

**Comment by zkywalker230061** [comment]
> Something I'm still missing is that there are changes to the source file, webplate will still consider unfinished and I haven't found a way to tell him: we don't care it's correct skip it. image Node is correct.

There should be a small tick box saying “require editing” or similar meaning at the left down corner. Once unticked weblate knows no further editing is required. No worries it’s all set for now.

**Pull request merged**